blackbox/docs/sql/geo.txt

```diff
@@ -151,29 +151,33 @@ we want to match. Available ``match_types`` are:
     cr> SELECT name from countries
```

Comment:
add introduction, e.g. 

assuming you have a table `countries` with a column `geo` of type `GEO_SHAPE INDEX USING geohash` ...